### PR TITLE
7840 - Added overnight event view when end time goes to next day

### LINKED ISSUE #7840: Week view: show time in over night shift
**Is your feature request related to a problem or use case? Please describe.**
Use case: if we are showing an over night shift in week view component it is not showing the time.
example: if the shedule is starting from 21-08-2023 11pm to 22-08-2023 1am then it directly shows its an over night shift which is fine but the duration is not shown

**Describe the solution you'd like**
Required time duration on over night shifts displayed on week view and if the name is big it is limiting characters

**Additional context**
image
image

### PR DESCRIPTION
**Pull request by yohannahbautista**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Added overnight event view when end time goes to next day

**Related github/jira issue (required)**:

Closes #7840 

**Steps necessary to review your pull request (required)**:

- Pull this branch, build and run the app
- Go to: http://localhost:4000/components/week-view/example-stacked.html
- Overnight event (gray, "Holiday Trip") should not be in all day row and should be in 22 and 23

**Included in this Pull Request**:
- [ ] A note to the change log.

### CODE REVIEW

**Review by tmcconechy — CHANGES_REQUESTED** [review]
@yohannahbautista the only problem with this is this was for the stacked example http://localhost:4000/components/week-view/example-stacked.html

So it makes it look confusing in that example because the time doesnt line up right. Can you change the new example so it shows a stacked example? Thanks!

**Comment by tmcconechy** [comment]
@yohannahbautista http://localhost:4000/components/week-view/example-stacked.html looks good. Maybe we just dont need http://localhost:4000/components/week-view/test-overnight.html at all as it is confusing that the times dont line up.

If you agree just delete it and i think we are good 👍🏻

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -66,6 +66,7 @@
 - `[Toolbar]` Added additional selectors and colors for dark theme dropdown label. ([#7897](https://github.com/infor-design/enterprise/issues/7897))
 - `[Tree]` Fixed Cross-Site Scripting (XSS) when setting up tree node. ([#7631](https://github.com/infor-design/enterprise/issues/7631))
 - `[Weekview]` Adjusted the positioning of text within the footer cell to keep it centered. Adjusted calendar icon position to be better aligned. ([#7926](https://github.com/infor-design/enterprise/issues/7926))
+- `[Weekview]` Added overnight event view when end time goes to next day. ([#7840](https://github.com/infor-design/enterprise/issues/7840))
```
Comment (ericangeles) [inline docs/CHANGELOG.md:69]: Please add this to 4.89.0 log @yohannahbautista

**Review by tmcconechy — DISMISSED** [review]
Aside from the change log approving

**Review by tjamesallen15 — APPROVED** [review]

**Comment by max-hizon** [comment]
Works on Chrome, Edge, and Firefox browsers in both New and Classic versions
image

image

**Review by glenlieorillo — DISMISSED** [review]
image

image


image

**Review by janahintal — APPROVED** [review]
Approving 
image

**Review by jbrcna — APPROVED** [review]
image

**Comment by jbrcna** [comment]
in mobile device, overnight event is not visible (23rd date)
image
image

**Pull request merged**